frontend/src/main/scala/akkaviz/frontend/FrontendUtil.scala

```diff
@@ -75,4 +78,32 @@ trait FancyColors {
     possibleColors(Math.abs(str.hashCode) % possibleColors.size)
   }
 
+  def hsl(hue: Double, saturation: Double, lightness: Double): Color = {
+    val c: Double = 1 - abs(2 * lightness - 1) * saturation
+    val x: Double = c * (1 - abs(((hue / 60) % 2) - 1))
+    val m: Double = lightness - c / 2
+
+    val (r: Double, g: Double, b: Double) =
+      if (hue < 60) (c, x, 0)
+      else if (hue < 120) (x, c, 0)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using case matching instead of else if blocks](https://www.codacy.com/app/lustefaniak/akka-viz/file/2290254515/issues/source?bid=3121282&fileBranchId=3189757#l88)